### PR TITLE
added Asked by Open AI no. 366

### PR DESCRIPTION
**Pull request by Almoeeni**

### CODE REVIEW

File: Solutions/366.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+if __name__ == "main ":
+n = int(input())
+m = int(input())
+mtx = [[int(val) for val in pair.split()]
+       for pair in input().strip().split(', ')]
+output = solutions(n, m, mtx)
+if output == True:
+    print("true")
+else:
+    print("false")
```
Comment (ruppysuppy) [inline Solutions/366.py]: Not indented

File: Solutions/366.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+                played.add((player2, player1))
+    for x in range(1, n+1):
+        for y in range(x+1, n+1):
+            if (x, y) not in played:
+                return False
+    return True
+
+
+# README
+# DO NOT CHANGE the code below, we use it to grade your submission. If changed your submission will be failed
```
Comment (ruppysuppy) [inline Solutions/366.py]: stray comments

**Review by ruppysuppy — CHANGES_REQUESTED** [review]
The repo follows [black convention](https://pypi.org/project/black/), so make sure it's formatted properly.

I would be travelling for the next 5 days, so I might be slow to respond

File: Solutions/366.py (hunk 2 above)
Comment (Almoeeni) [reply Solutions/366.py]: > stray comments

I fix these issues highlighted by you.

**Comment by ruppysuppy** [comment]
Thanks for making the changes, but from [here](https://github.com/dhanendraverma/Daily-Coding-Problem#problem-366) it seems like the problem is completely different?

**Comment by Almoeeni** [comment]
> Thanks for making the changes, but from [here](https://github.com/dhanendraverma/Daily-Coding-Problem#problem-366) it seems like the problem is completely different?

Can you please elaborate me please. you mentioned different repo instead of [this](https://github.com/ruppysuppy/Daily-Coding-Problem-Solutions)

**Comment by ruppysuppy** [comment]
As in the PR is pretty much ready to be merged, but are you sure that the problem you shared is the Daily Coding Problem 366?

**Comment by Almoeeni** [comment]
> As in the PR is pretty much ready to be merged, but are you sure that the problem you shared is the Daily Coding Problem 366?

I know that it's going to be 1 day up from 365 day but the main thing is that it's from Open AI and the repo is purely python based solutions, So that's why i'm contributing to it.

**Pull request merged**